crates/node-maintainer/tests/resolver_basic.rs

```diff
@@ -203,7 +204,8 @@ async fn nesting_sibling_conflict() -> Result<()> {
         .resolve_spec("a@^1")
         .await?;
 
-    nm.write_lockfile("./package-lock.kdl").await?;
+    let lockfile_path = "tests/package-lock.kdl";
+    nm.write_lockfile(lockfile_path).await?;
```

Comment:
looking at this now, I think this is a leftover from trying to debug issues in the resolver. I think you can just remove this `write_lockfile` completely and not have to worry about deleting it.